### ISSUE TITLE
Detailed param setting in Office-31

### ISSUE THREAD

**Issue by KaiyangZhou** [ChrisAllenMing/LtC-MSDA#2]
Hi,

Could you provide a more detailed training setup of using AlexNet in office-31? E.g., learning rate decay, data augmentation.

Table 6 in the supplementary is helpful but not sufficient to reproduce the results, especially on the Amazon domain where I could only get around 41% accuracy, which is far from the 51.6% accuracy as reported for SourceOnly.

Btw, I used a constant learning rate of 5e-5, and data augmentation of random flip & crop.

Thanks

**Comment by 1061136002** [comment]
> Hi,
> 
> Could you provide a more detailed training setup of using AlexNet in office-31? E.g., learning rate decay, data augmentation.
> 
> Table 6 in the supplementary is helpful but not sufficient to reproduce the results, especially on the Amazon domain where I could only get around 41% accuracy, which is far from the 51.6% accuracy as reported for SourceOnly.
> 
> Btw, I used a constant learning rate of 5e-5, and data augmentation of random flip & crop.
> 
> Thanks
I only get around 43% accuracy.

**Comment by YugeTen** [comment]
Running into the same issue!
